### ISSUE TITLE
Disabling Temando_Shipping results in PHP Fatal Error

### ISSUE THREAD

**Issue by DigitalStartupUK** [magento/magento2#27244]
### Preconditions (*)

1. Magento 2.3.4
2. Developer Mode

### Steps to reproduce (*)

1. Disabled Temando Shipping (`bin/magento module:disable Temando_Shipping`)
2. Compile (`bin/magento setup:di:compile`)
3. Clean and Flush Cache (`bin/magento c:c && bin/magento c:f`)
4. Add New Product from Backend

### Expected result (*)

1. Page loads and allows Admin to add a new products

### Actual result (*)

1. Blank Page
2. Error logged in Apache

```
PHP Fatal error:  Uncaught Error: Cannot instantiate interface Temando\\Shipping\\Model\\ResourceModel\\Repository\\PackagingRepositoryInterface in /var/www/html/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:50
```

If there is a dependency that requires `Temando_Shipping`, then the ability to disable it should not be an option.

**Comment by mattijv** [comment]
@DigitalStartupUK I don't think any of the core code depends on the Temando module. Did you run the setup:di:compile command (or delete the generated folder) after disabling the module? To me that seems like your dependency injection cache still contains references to the Temando classes.

**Comment by DigitalStartupUK** [comment]
I did a quick sanity check, resulting in the same issue:
```
bin/magento module:disable Temando_Shipping
bin/magento setup:di:compile
bin/magento c:c && bin/magento c:f
```

I should also note, that I'm sitting in Dev Mode. I've updated the original issue to include this.

**Comment by starlyns** [comment]
Another problem is that you can't import or export products while Temando is disabled.

**Comment by lmstephenson** [comment]
> Another problem is that you can't import or export products while Temando is disabled.

I have confirmed this to be true as well.

1. Fresh install of Magento 2.3.4 from CLI
2. Disable Temando from CLI (php bin/magento module:disable Temando_Shipping)
3. Clear caches, upgrade, compile, deploy... yada yada yada
4. Using `System > Data Transfer > Import (Entity type: Products, Import Behavior: Add/Update, Validation Strategy: Stop on Error) will not validate, therefore cannot complete

[POST] 500 error `[...]/adminhtml/Magento/backend/en_US/mage/backend/validation.js`

**Comment by DominicWatts** [comment]
More specifically at seeing exception when editing products.

Temando_Shipping module extension might have added an EAV attribute already with a specific source model that no longer exists. This could generate an exception Class Temando\Shipping\Model\Source\PackagingType does not exist. To fix this, run the following SQL query in your database:

DELETE FROM `eav_attribute` WHERE `source_model` LIKE '%Temando%'

**Comment by damienwebdev** [comment]
I'm seeing this on v2.3.4 just trying to install a fresh Magento instance. The module does not exist in my `config.php`, but it still runs the migrations:

```
[Progress: 246 / 994]
Module 'Temando_Shipping':
```

Finally crashing with:

```
Fatal error: Uncaught Error: Cannot instantiate interface Temando\Shipping\Model\Config\ModuleConfigInterface in /var/www/html/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php:121
Stack trace:
#0 /var/www/html/vendor/magento/framework/ObjectManager/Factory/Compiled.php(108): Magento\Framework\ObjectManager\Factory\AbstractFactory->createObject('Temando\\Shippin...', Array)
#1 /var/www/html/vendor/magento/framework/ObjectManager/Factory/Compiled.php(150): Magento\Framework\ObjectManager\Factory\Compiled->create('Temando\\Shippin...')
#2 /var/www/html/vendor/magento/framework/ObjectManager/Factory/Compiled.php(79): Magento\Framework\ObjectManager\Factory\Compiled->get('Temando\\Shippin...')
#3 /var/www/html/vendor/magento/framework/ObjectManager/Factory/Compiled.php(150): Magento\Framework\ObjectManager\Factory\Compiled->create('Temando\\Shippin...')
#4 /var/www/html/vendor/magento/framework/ObjectManager/Factory/Compiled.php(79): Magento\Framework\ObjectManager\Factory\Compiled->get('Temando\\Shippin...') in /var/www/html/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php on line 121
```

**Comment by pmarki** [comment]
Temando installs couple of product attributes. So once the setup scrips were fired those have to be deleted before temando can be disabled/removed
 'ts_dimensions_length', 'ts_dimensions_width', 'ts_dimensions_height', 'ts_packaging_type',  'ts_packaging_id' 'ts_hs_code'.

**Comment by starlyns** [comment]
Ok for the rest of us not geniuses. How to delete product attributes that can't be deleted from the admin side?

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @sudheers-kensium 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-32687` were created

**Issue Available**: @sudheers-kensium, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by robocrax** [comment]
> Ok for the rest of us not geniuses. How to delete product attributes that can't be deleted from the admin side?

SQL Table `eav_attribute` remove entries starting with `ts_` (or search for Temando)

https://magento.stackexchange.com/a/301103

**Comment by magento-engcom-team** [comment]
Hi @DigitalStartupUK. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-32687` was closed as `non-reproducible` in `2.4-develop`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean Magento instance or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.
